mula/tests/integration/test_listeners.py

```diff
@@ -0,0 +1,216 @@
+import threading
+import time
+import unittest
+from unittest import mock
+
+import pika
+from scheduler import connectors, utils
+
+from tests.mocks import listener
+
+
+class RabbitMQTestCase(unittest.TestCase):
+    def setUp(self):
+        self.dsn = "amqp://guest:guest@ci_rabbitmq:5672/%2Fkat"
```

Comment:
This could be a constant out of the class, or a class constant, as it's not a class instance variable that needs to be instantiated for each test